lambdas/helpers.py

```diff
@@ -0,0 +1,51 @@
+import base64
+import hmac
+import json
+import os
+
+import requests
+
+SAMPLE_WEBHOOK_POST_BODY = {
```

Comment:
Non-blocking recommendation, if these constants and functions are just needws for local/dev testing, I would recommend a name other than `helpers` since we normally seem to use that for functions that are needed for regular operations but not easily categorized.